### ISSUE TITLE
MVC Can't Register Session 

### ISSUE THREAD

**Issue by live-xy** [kataras/iris#1197]
Iris Version : 11.2 And 11.1
Go Version : go version go1.11.5 darwin/amd64
Problem : Whether 11.2 or 11.1 , When use mvcapp.Configure(Session.Start) , it does't bind correct.
Code:
```
package main

import (
	"./web/controllers"
	"github.com/kataras/iris"
	"github.com/kataras/iris/mvc"
	"github.com/kataras/iris/sessions"

)
func main() {
	app := iris.New()
	app.Logger().SetLevel("debug")
	app.RegisterView(iris.HTML("./web/views", ".js"))
	// mvc.New(app.Party("/movies")).Handle(new(controllers.MovieController))
	teemo_report_session := sessions.New(sessions.Config{Cookie: "TeemReport"})

	teemo_report := mvc.New(app.Party("/report"))  

	teemo_report.Register(teemo_report_session.Start)
	teemo_report.Configure(report)
	fmt.Println(teemo_report)

	app.Run(
	iris.Addr("localhost:8080"),
	//iris.WithoutVersionChecker,
	iris.WithOptimizations)
}

```

`package controllers

import (
	"github.com/kataras/iris/mvc"
	"github.com/kataras/iris"
	"../../services"
	"github.com/kataras/iris/sessions"

	"fmt"
)

type ReportController struct {
	Session *sessions.Session

	Service services.ReportService
	Ctx iris.Context
}`

Bug:
![image.png](https://2up.ooo/file/get/124/image.png)

Attention:
When I Use example/mvc/session-controller to find correct method,
It Also show this 
seems it does't bind correctly  to ` type ReportController struct 's Session *sessions.Session`

**Comment by kataras** [comment]
Hello @live-xy , at `11.1` it works fine, the problem was on the work-in-progress `v11.2` because I've added variadic `context.CookieOption` slice to be passed at `sessions#Start` to customize the session's cookie as requested at: #1186 , this is fixed now on [v11.2](#1175). Thank you!

**Comment by kataras** [comment]
Tip: you can set the logger to `"debug"` level to see what controller's fields are binded before serve-time to reduce issues like that on your screen,  `app.Logger().SetLevel("debug")` 
![see](https://user-images.githubusercontent.com/22900943/52914367-0d587c80-32d0-11e9-873d-5cd7823e855a.png)
